x-pack/legacy/plugins/apm/server/lib/errors/get_error_groups.ts

```diff
@@ -112,8 +112,8 @@ export async function getErrorGroups({
       return {
         message,
         occurrenceCount: bucket.doc_count,
-        culprit: idx(source, _ => _.error.culprit),
-        groupId: idx(source, _ => _.error.grouping_key),
+        culprit: source.error.culprit,
```

Comment:
type says both `culprit` and `grouping_key` are always defined. don't want to use `idx` because it makes the property optional, which causes issues for consumers.

Reply:
Great! 👍 We shouldn't use `idx` for non-optional properties anyway